src/App.js

```diff
@@ -1,21 +1,23 @@
-import React, { Suspense, lazy } from 'react';
+import React, { Suspense, lazy } from "react";
```

Comment:
Please resolve conflicts with master branch in App.js

Reply:
Resolved Conflicts. Can be merged now

Reply:
Merging....